src/hb-ot-layout.cc

```diff
@@ -275,12 +277,15 @@ hb_ot_layout_get_ligature_carets (hb_font_t      *font,
 				  unsigned int   *caret_count /* IN/OUT */,
 				  hb_position_t  *caret_array /* OUT */)
 {
-  return font->face->table.GDEF->table->get_lig_carets (font,
-							direction,
-							glyph,
-							start_offset,
-							caret_count,
-							caret_array);
+  unsigned int result_caret_count = 0;
```

Comment:
You don't need this?  Just pass caret_count to both APIs.

Reply:
Then `GDEF` makes it zero and `lcar` can't fill it.

Reply:
Got it.  You're right.